### PR TITLE
feat-ios-sdk

### PR DESCRIPTION
**Pull request by IloveJavaa**
feat-ios-sdk
Link - https://app.asana.com/0/0/1203232843311351/f

### CODE REVIEW

File: Assets/Trackier/iOS/TrackieriOS.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+using UnityEditor.PackageManager;
+using UnityEngine;
+
+
+namespace com.trackier.sdk
+{
+#if UNITY_IOS
+	public class TrackieriOS
+	{
+#if UNITY_IOS
```
Comment (Hemant-Mann) [inline Assets/Trackier/iOS/TrackieriOS.cs:14]: this if can be removed because there is an if above @IloveJavaa

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: Assets/Trackier/iOS/TrackieriOS.cs (hunk 1 above)
Comment (IloveJavaa) [reply Assets/Trackier/iOS/TrackieriOS.cs:13]: Done

File: Assets/Trackier/Android/TrackierAndroid.cs [hunk 2]
```diff
@@ -88,6 +88,32 @@ public static void setUserPhone(string userPhone)
...
 
+	public static void setUserEmail(string userEmail)
+	{
+		try {
+			AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
+			AndroidJavaObject trackierSDK = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+			AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");	   
+			trackierSDK.CallStatic("setUserEmail",userEmail);  
+		}
+		catch(System.Exception e){
```
Comment (Hemant-Mann) [inline Assets/Trackier/Android/TrackierAndroid.cs:99]:
```suggestion
		catch(System.Exception e) {
```

File: Assets/Trackier/Android/TrackierAndroid.cs [hunk 3]
```diff
@@ -88,6 +88,32 @@ public static void setUserPhone(string userPhone)
...
+
+	public static void setUserId(string userId)
+	{
+		try {
+			AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
+			AndroidJavaObject trackierSDK = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+			AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");	   
+			trackierSDK.CallStatic("setUserId",userId);  
+		}
+		catch(System.Exception e){
```
Comment (Hemant-Mann) [inline Assets/Trackier/Android/TrackierAndroid.cs:112]:
```suggestion
		catch(System.Exception e) {
```

Comment (IloveJavaa) [reply Assets/Trackier/Android/TrackierAndroid.cs]: Done

File: Assets/Trackier/Android/TrackierAndroid.cs (hunk 2 above)
Comment (IloveJavaa) [reply Assets/Trackier/Android/TrackierAndroid.cs]: Done

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**